src/modules/real-time-metrics/reports/convert-beholder-to-chart.js

```diff
@@ -345,6 +370,64 @@ const formatMapChartData = ({ report, data }) => {
   ]
 }
 
+const formatStackedAreaChart = ({ report, data }) => {
+  const dataset = Object.keys(data)
+
+  const timestamps = [
+    ...new Set(
+      data[dataset]
+        .map((entry) => {
+          const date = new Date(entry[report.xAxis])
+          return date.toISOString().split('T')[0] // Returns this date format: 'YYYY-MM-DD'
```

Comment:
melhore o texto:
Sugiro remover o comentário e tornar o código mais claro:
```suggestion
          const entryDate = new Date(entry[report.xAxis]);
          const formattedDate = date.toLocaleDateString('en-CA')
         return formattedDate
```
OR
```suggestion
          const entryDate = new Date(entry[report.xAxis]);
          const formattedDate = entryDate.toISOString().split('T')[0];
          return formattedDate;
```